.github/workflows/shellcheck.yml

```diff
@@ -0,0 +1,23 @@
+---
+name: Shellcheck
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  shellcheck:
+    name: Check shell scripts
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install dependencies
+        run: |
+          sudo apt update && sudo apt install -y shellcheck
+      - name: shellcheck
+        run: |
+          git grep -l '^#\( *shellcheck \|!\(/bin/\|/usr/bin/env \)\(sh\|bash\|dash\|ksh\)\)' | xargs shellcheck
```

Comment:
I have put all the tests in the [tests.yml](https://github.com/oxigraph/oxigraph/blob/main/.github/workflows/tests.yml) file. Could you add it there too?

Reply:
done!